.github/pull_request_template.md

```diff
@@ -1,17 +1,20 @@
-## What is this PR doing?
+# What is this PR doing?
 
 
-## How can these changes be manually tested?
+# How can these changes be manually tested?
 
 
-## Does this PR resolve or contribute to any issues?
+# Does this PR resolve or contribute to any issues?
 
 
-## Checklist
+# Checklist
 - [ ] I have manually tested these changes
 - [ ] Post a link to the PR in the group chat
 
-## Guidelines
+# Guidelines
+- Sometimes PRs are a source of conflict, please be mindful of your colleagues
```

Comment:
Since we would like to accept/be open to external contributions asides from the core team, I would word this differently.

```suggestion
- Sometimes PRs can cause conflicts with other in flight changes. Be sure to collaborate with other authors when this occurs.
```

Reply:
Perhaps this wording:

```md
- Disagreements happen in PRs, please be careful how you handle these and be mindful of others
```

Reply:
Updated suggestion